### PR TITLE
Add kubebuilder print-columns for database, cluster, and workload types

### PR DESCRIPTION
**Pull request by lukeweber**
Resolves: https://github.com/crossplaneio/crossplane/issues/38

### CODE REVIEW

**Comment by jbw976** [comment]
please amend your commit to include the DCO sign-off also

**Comment by lukeweber** [comment]
Would like to make a pass and decide which fields we want. i.e. would we like AGE on all elements? 

This requires k8s 1.11, so if we update the demo docs we would need 1.11 as a requirement to run the demo.

Any thoughts?

**Comment by jbw976** [comment]
I'd like to get this into master sooner than later so that we can clean up our docs.  I think the docs clean-up can be done in a later PR.

I'd like to add an Age column, can we do that easily? I don't think we specify anywhere in the docs that we require k8s 1.11+ but what happens if the target k8s cluster is not that minimum version? Does the age column show up empty, or does the whole `kubectl get` break and return an error?

**Comment by lukeweber** [comment]
It just doesn't seem to parse the extra data about labels in CRDs in k8s 1.10. Until 1.11 is more widespread we might wait on removing the labels from the kubectl commands in our docs.

File: pkg/apis/gcp/database/v1alpha1/cloudsql_instance_types.go [hunk 1]
```diff
@@ -76,6 +76,11 @@ type CloudsqlInstanceStatus struct {
 // CloudsqlInstance is the Schema for the instances API
 // +k8s:openapi-gen=true
 // +groupName=database.gcp
+// +kubebuilder:printcolumn:name="NAME",type="string",JSONPath=".metadata.name"
+// +kubebuilder:printcolumn:name="STATUS",type="string",JSONPath=".status.state"
+// +kubebuilder:printcolumn:name="CLASS",type="string",JSONPath=".spec.classRef.name"
+// +kubebuilder:printcolumn:name="VERSION",type="string",JSONPath=".spec.version"
```
Comment (jbw976) [inline pkg/apis/gcp/database/v1alpha1/cloudsql_instance_types.go]: i think this should actually be:
```
// +kubebuilder:printcolumn:name="VERSION",type="string",JSONPath=".spec.databaseVersion"
```

File: pkg/apis/aws/database/v1alpha1/rdsinstance_types.go [hunk 2]
```diff
@@ -86,6 +86,11 @@ type RDSInstanceStatus struct {
 // RDSInstance is the Schema for the instances API
 // +k8s:openapi-gen=true
 // +groupName=database.aws
+// +kubebuilder:printcolumn:name="NAME",type="string",JSONPath=".metadata.name"
+// +kubebuilder:printcolumn:name="STATUS",type="string",JSONPath=".status.state"
+// +kubebuilder:printcolumn:name="CLASS",type="string",JSONPath=".spec.classRef.name"
+// +kubebuilder:printcolumn:name="VERSION",type="string",JSONPath=".spec.version"
```
Comment (jbw976) [inline pkg/apis/aws/database/v1alpha1/rdsinstance_types.go]: i think this should actually be:
```
// +kubebuilder:printcolumn:name="VERSION",type="string",JSONPath=".spec.engineVersion"
```

**Comment by lukeweber** [comment]
@jbw976- resolved the issues you raised, removing name and updating the incorrect references. I also added "-" between words as we had a mix in our code.

**Comment by jbw976** [comment]
Thanks for this contribution and following up @lukeweber!

**Pull request merged**